pkg/kubelet/server/server.go

```diff
@@ -878,6 +891,52 @@ func (s *Server) getPortForward(request *restful.Request, response *restful.Resp
 	proxyStream(response.ResponseWriter, request.Request, url)
 }
 
+// checkpoint handles the checkpoint API request. It checks if the requested
+// podNamespace, pod and container actually exist and only then calls out
+// to the runtime to actually checkpoint the container.
+func (s *Server) checkpoint(request *restful.Request, response *restful.Response) {
+	if request == nil || response == nil {
+		return
+	}
+	options := &runtimeapi.CheckpointRestoreOptions{}
+	if request.Request.ContentLength > 0 {
+		if err := request.ReadEntity(options); err != nil {
+			response.WriteError(
+				http.StatusInternalServerError,
+				fmt.Errorf("failed to parse POST parameters (%v)", err),
+			)
+			return
+		}
+	}
+	klog.InfoS("Request PAram login", "options", options)
```

Comment:
this log line should be at a certain log level.

Reply:
I removed that line, that was a debug message left over and not necessary any more.